### PR TITLE
Pet bio page new

### PR DESCRIPTION
**Pull request by mnca16**
In this PR:

- Added responsive layout

### CODE REVIEW

**Review by tarikutk — APPROVED** [review]
Looks Good.

File: src/pages/PetBio.js [hunk 1]
```diff
@@ -27,197 +28,270 @@ function PetBio() {
...
+            md: "800px",
+            lg: "850px",
+            xl: "1000px",
+          },
+          margin: "0 auto",
+          marginTop: "50px",
+        }}
+      >
+        <Typography variant="h3" component="div">
+          My namae is {findPetObject.name}!
```
Comment (IreneS1) [inline src/pages/PetBio.js:47]: Name is miss spelled

**Comment by IreneS1** [comment]
Like Tariku said, I think the last part "i'm good with:" gets lost. Maybe add more separation to that section or do as Tariku suggested and specify in each line.

File: src/pages/PetBio.js (hunk 1 above)
Comment (mnca16) [reply src/pages/PetBio.js]: I did not see that, thank you so much!! just fixed it

**Comment by mnca16** [comment]
Just added some of the suggestions!

**Review by IreneS1 — APPROVED** [review]

**Review by MarissaLZ — APPROVED** [review]

**Pull request merged**